adapters/sidekiq/lib/opentelemetry/adapters/sidekiq/middlewares/client/tracer_middleware.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+# Copyright 2020 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Adapters
+    module Sidekiq
+      module Middlewares
+        module Client
+          class TracerMiddleware
+            def call(worker_class, job, _queue, _redis_pool)
+              tracer.in_span(
+                worker_class,
+                attributes: {
+                  jid: job['jid'],
+                  created_at: job['created_at'],
```

Comment:
Technically, this shouldn't be required. Trace analysis should be able to infer it from the parent span (the producer).

Reply:
Uh, well, that's embarrassing -- this *is* the producer. Do we have the `created_at` at this point? Does it matter? Is the span start time a reasonable substitute?

Reply:
>Is the span start time a reasonable substitute?

I think it is reasonable, not sure if it's worth splitting hairs here but the span start time would be after the actual `created_at` timestamp.  

Theres another time value that could provide some value, which is the `enqueued_at` time.  It's available in the server middleware, but also on client middleware when using the sidekiq activejob adapter.

Reply:
Hrm... Why not both? Let's add both `created_at` and `enqueued_at`.